src/Cleave.js

```diff
@@ -111,6 +116,23 @@ Cleave.prototype = {
         this.onInput(this.element.value);
     },
 
+    onCutOrCopy: function(e) {
+        var owner = this, 
+            pps = owner.properties,
+            Util = Cleave.Util,
+            inputValue = owner.element.value,
+            copyDelimiter = owner.properties.copyDelimiter,
+            textToCopy = '';
+
+        if (!copyDelimiter) {
```

Comment:
This variable is only used once, so not necessary to assign a new variable for it. Can just use: pps.copyDelimiter here ;)